contents/blog/cookieless-analytics.md

```diff
@@ -0,0 +1,41 @@
+---
+date: 2021-08-13
+title: Cookieless Analytics - How To Track Website and App Users Without Third-Party Cookies
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: marcus-hyett
+featuredImage: ../images/blog/filename.png
+featuredImageType: full
+---
+
+Google has stated that they intend to deprecate the use of 3rd party cookies in 2023. But why is this important, and how will this affect product analytics?
+
+## What are 3rd party cookies?
+
+Cookies are small files stored on the user’s machine, helping you identify a repeat user to your product or website. This makes things like staying logged in over multiple days possible, but it also makes it possible to track whether the person who followed an ad yesterday came back today to purchase something.
+3rd party cookies are a particular flavour of cookie provided by an external service and not your website directly. This external service can be a tool or marketing product that you work with to optimize ads.
+
+## What happens when third-party cookies go away?
+
+When 3rd party cookies are deprecated, any external tools you use in your product or website to track users - either for analytics or marketing purposes - will cease to work correctly.
+Every visit to your website from the same user will look the same as a brand new user, making it difficult to attribute outcomes to your ad campaigns or understand whether a single person is avidly using your product or it’s a number of users using it casually.
+
+## What are the alternative solutions?
+
+It’s key to note that if people are logged into your product or website - you should still be able to track them (with any tool) - since you’ll be relying on first party cookies.
```

Comment:
@marcushyett-ph I'd argue this is quite confusing - most logged in users today are tracked by third party cookies, from Amplitude/Heap etc

Reply:
As above - I don't believe that's the case.

Reply:
What Marcus says is definitely possible, but I don't think this is the case by default? (much more complicated than the out-of-the-box solutions provided by Heap or Amplitude) - You'd have to implement this yourself.

By default, these libraries (even posthog-js for Cloud) generates a 3rd party cookie - for first-timers and logged in users both.

What's different / cool about PostHog is that this automated 3rd party cookie generation becomes 1st party when you're self-hosting.

Reply:
Ah okay - so to clarify - if you're using a product with 3rd party cookie tracking - you'd have to do a bunch of work to get it to work with logged in users as well as logged out users? But you wouldn't if you used a self-hosted (1st party cookie) solution?

Reply:
You'd have to do a bunch of work to get it to work with **logged in users and converting the cookies to 1st party**. By default, 3rd party cookies work for both kinds of users. It's the converting-to-1st-party that is non-trivial bit.